packages/manager/src/managers/prismicRepository/types.ts

```diff
@@ -155,3 +155,25 @@ export type StarterId =
 	| "nuxt_multi_page"
 	| "nuxt_blog"
 	| "nuxt_multi_lang";
+
+export const Environment = t.type({
```

Comment:
Ideally, I think we'd only want to type and check for the properties we need within Slice Machine. For example, if we aren't going to use `pendingUsers`, we could omit it from the type.

I think this would decouple Slice Machine file external libraries (like the Core API), making it easier to maintain with greater flexibility. It also would ensure we could continue using `io-ts` (like all other data validation in SM) without caring with the Core API uses.

What do you think?

Reply:
Perfect, I agree